### PR TITLE
data concept: requirements

### PR DESCRIPTION
**Pull request by d70-t**
This PR adds the requirements section to the data concept.

### CODE REVIEW

File: orcestra_book/data_concept.md [hunk 1]
```diff
@@ -28,3 +30,78 @@ The goals are sorted in decreasing priority (i.e. 1 is the most important). We *
...
+* You **should** follow standard metadata schemes ([CF-Conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html))
+* You **may** designate data processing levels to help users understand the quality of the provided data. If you provide processing levels, the levels **should** follow the [EOSDIS data processing levels](https://www.earthdata.nasa.gov/engage/open-data-services-and-software/data-information-policy/data-levels) scheme.
+* You **should** work with your own **published** datasets.
+* Datasets **should** be reviewed across teams.
+
+### 4. incremental backups are possible
+
+*(this section is currently in an early stage)*
+
+* currently we can't deliver a copy of "the EUREC4A data" <!-- (e.g. to deliver it to Barbados as promised) -->
```
Comment (tmieslinger) [inline orcestra_book/data_concept.md:87]: I'd leave this point out as it is not really linked to ORCESTRA and people that didn't follow the EUREC4A data discussions won't understand the point. I'd say this point is covered by the following points and worst case, it's confusing to the reader.

File: orcestra_book/data_concept.md [hunk 2]
```diff
@@ -28,3 +30,78 @@ The goals are sorted in decreasing priority (i.e. 1 is the most important). We *
...
+* storage location has to provide a method to check if something changed
+    * e.g. HTTP ETag, hash of content etc...
+* how to handle updates / changes to adataset?
```
Comment (tmieslinger) [inline orcestra_book/data_concept.md:90]:
```suggestion
* how to handle updates / changes to a dataset?
```

**Review by tmieslinger — COMMENTED** [review]
The requirements are a good start for further discussions! Thanks for putting that online. It will help to distribute, discuss and improve the data concept, also towards possible implementations.

File: orcestra_book/data_concept.md (hunk 1 above)
Comment (d70-t) [reply orcestra_book/data_concept.md]: 👍 I removed it

**Pull request merged**